holoviews/util/settings.py

```diff
@@ -361,7 +361,7 @@ def update_options(cls, options, items):
     @classmethod
     def initialize(cls, backend_list):
         cls.backend_list = backend_list
-        backend = cls.options.get('backend', Store.current_backend)
+        backend = Store.current_backend
```

Comment:
Sure. I can do that to check. That said, those switches shouldn't be going through ``initialize`` each time anyway.